ext/opentelemetry-ext-grpc/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+- Add status code to gRPC client, gRPC client testing
```

Comment:
"gRPC client testing" is unnescessary, as that doesn't affect the user directly.

Reply:
fixed, but it seems like the PR is not updating? (my branch has the right commits though?) might have to remake the PR